xml/System.Net.Http/HttpClient.xml

```diff
@@ -2536,8 +2695,10 @@ The size specified is greater than the maximum allowed buffer size.</exception>
 
  ]]></format>
         </remarks>
+        <exception cref="T:System.InvalidOperationException">The <paramref name="requestUri" /> must be an absolute URI or <see cref="P:System.Net.Http.HttpClient.BaseAddress" /> must be set.</exception>
+        <exception cref="T:System.Net.Http.HttpRequestException">The request failed due to an issue getting a valid HTTP response such as network connectivity failure, DNS failure, server certificate validation error or invalid server response. On .NET 8+ the reason is indicated by <see cref="P:System.Net.Http.HttpRequestException.HttpRequestError" /></exception>
+        <exception cref="T:System.OperationCanceledException">The cancellation token was canceled or the request failed due to timeout.</exception>
```

Comment:
What is the reason to drop "This exception is stored into the returned task"?

Reply:
HttpClient API docs follow a the nonstandard practice of documenting every stored exception for the async methods as a normal exception. Having the "This exception is stored into the returned task" line just for the `OperationCanceledException` felt odd.